### PR TITLE
Refac move basis data

### PR DESCRIPTION
**Pull request by johevemi**
## Description
Migrates basis-data and master-basis-data

## References

* #199

### CODE REVIEW

File: source/databricks/tests/integration/calculator/test_calculator_job.py [hunk 1]
```diff
@@ -79,7 +80,7 @@ def executed_calculation_job(
         f"{test_files_folder_path}/MeteringPointsPeriods.csv",
         header=True,
         schema=metering_point_period_schema,
-    )
+    ).withColumn("gln", lit("grid_access_provider"))
```
Comment (JesperGomes) [inline source/databricks/tests/integration/calculator/test_calculator_job.py:83]: consider writing this string as a constant at a share location

File: source/databricks/package/datamigration/migration_scripts/202301091300_Move_Basis_Data.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        match = re.search(
+            r"calculation-output/basis-data/(batch_id=\w{8}-\w{4}-\w{4}-\w{4}-\w{12})/time-series-(\w*)/(grid_area=\d{3})",
+            directory.name,
+        )
+        if match and directory.is_directory:
+            batch_id = match.group(1)
+            resolution = match.group(2)
+            grid_area = match.group(3)
+            current_directory_name = directory.name
+            print(current_directory_name)
```
Comment (lasrinnil) [inline source/databricks/package/datamigration/migration_scripts/202301091300_Move_Basis_Data.py:51]: remove?

Comment (johevemi) [reply source/databricks/package/datamigration/migration_scripts/202301091300_Move_Basis_Data.py:51]: removed

File: source/databricks/tests/integration/calculator/test_calculator_job.py (hunk 1 above)
Comment (johevemi) [reply source/databricks/tests/integration/calculator/test_calculator_job.py]: done

**Pull request merged**